ar4_moveit_config/config/ros2_controllers.yaml

```diff
@@ -24,7 +24,6 @@ arm_controller:
       - position
     state_interfaces:
       - position
-      - velocity
```

Comment:
Can we test if Gazebo still works after these changes?

Reply:
Yes, it is working.